ruby-builder-versions.json

```diff
@@ -18,8 +18,8 @@
   "jruby": [
     "9.1.17.0",
     "9.2.9.0", "9.2.10.0", "9.2.11.0", "9.2.11.1", "9.2.12.0", "9.2.13.0", "9.2.14.0", "9.2.15.0", "9.2.16.0", "9.2.17.0", "9.2.18.0", "9.2.19.0", "9.2.20.0", "9.2.20.1", "9.2.21.0",
-    "9.3.0.0", "9.3.1.0", "9.3.2.0", "9.3.3.0", "9.3.4.0", "9.3.6.0", "9.3.7.0", "9.3.8.0", "9.3.9.0", "9.3.10.0", "9.3.11.0",
-    "9.4.0.0", "9.4.1.0", "9.4.2.0", "9.4.3.0", "9.4.4.0",
+    "9.3.0.0", "9.3.1.0", "9.3.2.0", "9.3.3.0", "9.3.4.0", "9.3.6.0", "9.3.7.0", "9.3.8.0", "9.3.9.0", "9.3.10.0", "9.3.11.0", "9.3.13.0",
```

Comment:
Note: there is no 9.3.12.0: https://github.com/jruby/jruby/releases/tag/9.3.13.0